contrib/mkinitcpio/sbctl

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/bash
-echo "Signing EFI binaries..."
-/usr/bin/sbctl sign-all -g
+
+KERENEL_FILE="$1"
+UKI_FILE="$3"
+
+IMAGE_FILE="$KERENEL_FILE"
+if [ -n "$KERNELDESTINATION" ] && [ -f "$KERNELDESTINATION" ]; then
+    IMAGE_FILE="$KERNELDESTINATION"
+fi
+if [ -n "$UKI_FILE" ]; then
+    IMAGE_FILE="$UKI_FILE"
+fi
+
+if [ -z "$IMAGE_FILE" ]; then
+    echo "No kernel or UKI found for signing"
+    exit 0
+fi
+
+echo "Signing $IMAGE_FILE"
+sbctl sign -s "$IMAGE_FILE"
```

Comment:
do we really need `-s` here?

Reply:
Well, this has already been merged, but I think the `-s` is right here. Initramfs files should always be saved to the database and resigned as soon as possible.

Reply:
problem is they're never removed from the sbctl "database", so if you remove the uki, now the sbctl database is out-of-sync

Reply:
In that case, sbctl will just display an error, but the exit code will still be zero. So you can just remove from the database file manually when you see the message. That's why the current behavior is ok IMHO.